src/main/proto/th2_grpc_common/common.proto

```diff
@@ -154,6 +154,7 @@ enum FilterOperation {
   NOT_EQUAL = 1;
   EMPTY = 2;
   NOT_EMPTY = 3;
+  WILDCARD = 4;
```

Comment:
Maybe we should add a description that explains what is the `WILDCARD` and what are the limitations (or we should add this information to the README file at least)